### PR TITLE
Update qtFRED readme and add some documentation

### LINKED ISSUE #1696: Update qtFRED README.md

### PR DESCRIPTION
**Pull request by asarium**
This fixes #1696.

### CODE REVIEW

File: qtfred/README.md [hunk 1]
```diff
@@ -10,26 +10,83 @@ a glimpse of hope appeared to users of other operating systems.
...
-This project will follow the [SemVer](http://semver.org/) convention.
+### CMake configuration
+Configure the project as usual with CMake but make sure that `FSO_BUILD_QTFRED` is enabled in the CMake configuration. The
+variable `QT5_INSTALL_ROOT` is available for letting CMake know where it should look for the QT5 installation. This is
+especially important on Windows where the standard CMake `find_package` calls will likely fail.
+
+You need to set `QT5_INSTALL_ROOT` to the actual installation folder of a specific version. If you installed Qt to a path
+like `/path/to/Qt` then you need to set `QT5_INSTALL_ROOT` to `/path/to/Qt/<version>/<variant>` where `<version>` might be
+something like `5.9.2` and `<variant>` could be `msvc2017_64`. You need to make sure that the variant matches the compiler
+and bittness you are currently compiling FSO/qtFRED for. In the example above it was Visual Studio 2017 and FSO was built
```
Comment (niffiwan) [inline qtfred/README.md]: Should that be "bitness"?

File: qtfred/README.md [hunk 2]
```diff
@@ -10,26 +10,83 @@ a glimpse of hope appeared to users of other operating systems.
...
+
+Our CMake setup will automatically handle copying the relevant Qt DLLs to the build folder and if you use the CMake install
+functionality the DLLs will also be copied to the correct paths in the destination folder.
+
+Directories
+-----------
+
+- **resources**: Contains the Qt resources needed for qtFRED
+    - **images**: The image files used by qtFRED. If you add a new file make sure that it is also added to resources.qrc
+    - **win**: Contains Windows specific resources. Do not edito unless absolutely necessary.
```
Comment (niffiwan) [inline qtfred/README.md]: edit vs edito?

**Comment by asarium** [comment]
@niffiwan Both typos should be fixed now.

File: qtfred/README.md [hunk 3]
```diff
@@ -10,26 +10,83 @@ a glimpse of hope appeared to users of other operating systems.
...
+Build Instructions
+------------------
 
-  * CMake >= 2.8.12
-  * Qt5
+Additional Dependencies:
+
+- Qt >= 5.7
+
+qtFRED also needs all the depdencies that standard FSO needs.
```
Comment (tomimaki) [inline qtfred/README.md]: dependencies?

Comment (asarium) [reply qtfred/README.md]: A single word question isn't very useful. What do you want to say with this?

Comment (niffiwan) [reply qtfred/README.md]: I think it's just a typo; dependencies vs depdencies

Comment (asarium) [reply qtfred/README.md]: Oh, I didn't see that typo. Then this makes more sense.

Comment (asarium) [reply qtfred/README.md]: The typo should be fixed now.

**Pull request merged**